src/context/SettingsProvider.tsx

```diff
@@ -39,6 +50,9 @@ export const SettingsProvider = ({children}: {children: ReactNode}) => {
 
   const [soundEffect, setSoundEffect] = useState(localStorage.getItem('soundEffect') || 'None');
 
+  const storedCustomQRCodes = JSON.parse(localStorage.getItem('customQRCodes') || '{}');
```

Comment:
Just wondering, what's the reason for putting this in localStorage instead of adding it to the servers table in IndexedDB?

cc @tomasr8 since you're more familiar w/ the architecture of the app

Reply:
Without having looked at either PR too deeply, storing the custom qr code in the servers table together with all the other server data sounds better to me.

Also, if the local storage is cleared, there's a chance you'll end up with a broken state since the custom qr code data is only stored when you authenticate with the server for the first time.